cclib/parser/molcasparser.py

```diff
@@ -222,38 +222,41 @@ def extract(self, inputfile, line):
         #  Threshold at which QNR/C2DIIS is turned on 0.75E-01
         #  Threshold for Norm(delta) (QNR/C2DIIS)     0.20E-04
         if line[:34] == '++    Optimization specifications:':
-
-            scftargets = []
-            while not line[6:37] == 'Threshold for SCF energy change':
+            self.skip_lines(inputfile, ['d', 'b'])
+            line = next(inputfile)
+            if line.strip().startswith('SCF'):
+                scftargets = []
+                self.skip_lines(inputfile, ['Minimized', 'Number', \
+                                         'Maximum', 'Maximum'])
                 line = next(inputfile)
 
-            if line[6:37] == 'Threshold for SCF energy change':
-                target = float(line.split()[-1])
-                scftargets.append(target)
+                if line[6:37] == 'Threshold for SCF energy change':
```

Comment:
I question the robustness of using specific indices/slices when the string is so specific. It is unlikely to occur anywhere else, so maybe it is sufficient here to use `if 'very unique string' in line` in such cases?

Reply:
Yeah! Will make the change.